contracts/rust/gen-vk-contract/src/main.rs

```diff
@@ -42,7 +42,7 @@ fn main() {
     let (_, vk) = if mock {
         hotshot_state_prover::preprocess(&srs, 10).expect("Circuit preprocess failed")
     } else {
-        hotshot_state_prover::preprocess(&srs, STAKE_TABLE_CAPACITY)
```

Comment:
@mrain @alxiong Does it save significant amounts of computation to set capacity 10 for the demo? Or why are we doing it?

Reply:
I believe it brings down the proof generation time from ~2mins to <20s.